static/app/components/issueDiff/index.tsx

```diff
@@ -170,14 +180,20 @@ class IssueDiff extends Component<Props, State> {
         {loading && <LoadingIndicator />}
         {!loading &&
           DiffComponent &&
-          baseEvent.map((value, i) => (
-            <DiffComponent
-              key={i}
-              base={value}
-              target={targetEvent[i] ?? ''}
-              type="lines"
-            />
-          ))}
+          (this.state.newestFirst ? baseEvent.toReversed() : baseEvent).map(
+            (value, i) => (
+              <DiffComponent
+                key={i}
+                base={value}
+                target={
+                  this.state.newestFirst
+                    ? (targetEvent[targetEvent.length - 1 - i] ?? '')
```

Comment:
what exactly is this attempt to access. should targetEvent also be reversed? would that make it easier